sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
```

Comment:
Adding additional string processing is out of scope for this project.  We should stick to just exporting the javadoc strings and if additional processing is necessary it can be done after the fact.